ar-lib

```diff
@@ -0,0 +1,271 @@
+#! /bin/sh
```

Comment:
this should also be removed

Reply:
I put it on .gitignore to avoid futher commits involving it. Is it ok ?!